x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_details/monitor_errors/monitor_errors.tsx

```diff
@@ -33,15 +34,15 @@ export const MonitorErrors = () => {
   }
 
   return (
-    <>
+    <MonitorPendingWrapper>
```

Comment:
i will suggest instead of wrapping other content as children, we should try to render both in parallel. We render other content but keep it hidden. If condition becomes false that becomes visibile and loader goes hidden.